src/controller/testController.ts

```diff
@@ -290,7 +290,7 @@ async function getIncludedItems(request: TestRunRequest, token?: CancellationTok
  * @param testItems
  * @returns prepared testItems
  */
-function handleInvocations(testItems: TestItem[]): TestItem[] {
+export function handleInvocations(testItems: TestItem[]): TestItem[] { // export for unit test
```

Comment:
may be replaced by using [rewire](https://github.com/jhnns/rewire) ?

Reply:
Using `export` looks good to me now. We can consider using `rewire` if more cases are found.

BTW, we can add the comment `only export for unit test` to the TSDoc, so anyone tries to call it will see that.